public/injectEnv.js

```diff
@@ -4,6 +4,6 @@ window.injectedEnv ={
 	REACT_APP_USER_LOGOUT_URL:'https://k9dc.essential-dev.com/fence/logout',
 	REACT_APP_BACKEND_API: 'https://caninecommons-dev.cancer.gov/v1/graphql/',
 	REACT_APP_APPLICATION_VERSION:'YYYY_MM_DD/HH:MM',
-	REACT_APP_ABOUT_CONTENT_URL:'https://raw.githubusercontent.com/CBIIT/bento-icdc-frontend/master/src/content/dev/aboutPagesContent.yaml',
+	REACT_APP_ABOUT_CONTENT_URL:'https://raw.githubusercontent.com/CBIIT/bento-icdc-frontend/master/src/content/pre-prod/aboutPagesContent.yaml',
```

Comment:
In this case, we can remove dev stage and qa folders?

Reply:
Currently QA is having build and they are testing, at this point removing Dev and QA might break tat build. I can remove once build is passed from QA.

Reply:
Is it still the case?

Reply:
Updated PR.